lessons/css-intro/terminology.md

```diff
@@ -0,0 +1,24 @@
+| Attribute selectors | Match one or more elements based on their attributes/attribute values.                                                                                                                                                                                                                                                                       |
+| Pseudo-classes      | Match one or more elements that exist in a certain state, such as an element that is being hovered over by the mouse pointer, or a checkbox that is currently disabled or checked, or an element that is the first child of its parent in the DOM tree.                                                                                      |
+| Pseudo-elements     | Match one or more parts of content that are in a certain position in relation to an element, for example the first word of each paragraph, or generated content appearing just before an element.                                                                                                                                            |
+| Combinators         | These are not exactly selectors themselves, but ways of combining two or more selectors in useful ways for very specific selections. So for example, you could select only paragraphs that are direct descendants of `divs`, or paragraphs that come directly after headings.                                                                |
+| Multiple selectors  | Again, these are not separate selectors; the idea is that you can put multiple selectors on the same CSS rule, separated by commas, to apply a single set of declarations to all the elements selected by those selectors.                                                                                                                   |
+| em                  | 1em is the same as the font-size of the current element. But beware — font sizes are inherited by elements from their parents, so if different font sizes have been set on parent elements, the pixel equivalent of an em can start to become complicated.                                                                                   |
+| rem                 | The rem (root em) works in exactly the same way as the em, except that it will always equal the size of the default base font-size; inherited font sizes will have no effect, so this is a much better option than em                                                                                                                        |
+| vw, vh              | Respectively these are 1/100th of the width of the viewport, and 1/100th of the height of the viewport. Again, these are not as widely supported as em.                                                                                                                                                                                      |
```

Comment:
I would take out the part about "not as widely supported" - as a unit they are widely supported.

![Can I use  Support tables for HTML5, CSS3, etc 2020-09-14 09-08-44](https://user-images.githubusercontent.com/2607/93089840-e740bd00-f669-11ea-816c-480dd8c0be39.png)

Reply:
@gstark I got the definition from the handbook, should we fix that too?
https://handbook.suncoast.io/lessons/css-intro/values

Otherwise I fixed it on 34f492b68c2232a4e96ae0fb9e40437054127ef3

Reply:
Thanks! Send a PR for the handbook as well!